README.md

```diff
@@ -1,3 +1,337 @@
+-rw-rw-r--  1 terry  wheel  1344 Apr 11 15:25 EPI_ISL_601443-spike-aa-sequence.fasta
+-rw-rw-r--  1 terry  wheel  3886 Apr 11 15:25 EPI_ISL_601443-spike-nt-sequence.fasta
+```
+
+If the file given by `--genome` had contained more than one SARS-CoV-2
+genome, the spike would have been extracted for all of them. Similarly, if
+`--feature` had been repeated, multiple features would have been extracted
+and compared. If no `--feature` is given, you'll get them all.
+
+``` sh
+$ cat /tmp/out/EPI_ISL_601443-spike-aa-match.txt
+Feature: spike amino acid match
+  Matches: 1264/1274 (99.22%)
+  Mismatches: 10/1274 (0.78%)
+    Not involving gaps (i.e., conflicts): 7/1274 (0.55%)
+    Involving a gap in one sequence: 3/1274 (0.24%)
+    Involving a gap in both sequences: 0
+    Id: NC_045512.2 (surface glycoprotein)
+      Length: 1274
+      Gaps: 0
+    Id: EPI_ISL_601443 hCoV-19/England/MILK-9E05B3/2020 (surface glycoprotein)
+      Length: 1274
+      Gaps: 3/1274 (0.24%)
+      Gap locations (1-based): 69, 70, 144
+    Differences: site, aa1, aa2, ref nt codon start
+        69 H - 21767
+        70 V - 21770
+       144 Y - 21992
+       501 N Y 23063
+       570 A D 23270
+       614 D G 23402
+       681 P H 23603
+       716 T I 23708
+       982 S A 24506
+      1118 D H 24914
+```
+
+You can also ask for the changes in a variant to be checked.
+      
+``` sh
+$ describe-genome.py --genome data/EPI_ISL_601443.fasta --checkVariant VOC_20201201_UK
+EPI_ISL_601443 hCoV-19/England/MILK-9E05B3/2020
+Variant summary:
+  UK variant of concern (VOC) 202012/01:
+  20 checks, 20 passed.
+    orf1ab aa: PASS: 1001I, 1708D, 2230T, 3675-, 3676-, 3677-
+    spike aa: PASS: 1118H, 144-, 501Y, 570D, 681H, 69-, 70-, 716I, 982A
+    orf8 aa: PASS: 52I, 73C, Q27*
+    n aa: PASS: 235F, 3L
+```
+
+There are some known variants, and you can also provide your own via a JSON
+file and the `--variantFile` option. E.g.,:
+
+``` json
+VARIANTS = {
+    'VOC_20201201_UK': {
+        'description': 'UK variant of concern (VOC) 202012/01',
+        'comment': ('From Table 1 of https://www.gov.uk/government/'
+                    'publications/investigation-of-novel-sars-cov-2'
+                    '-variant-variant-of-concern-20201201'),
+        'changes': {
+            'orf1ab': {
+                'aa': '1001I 1708D 2230T 3675- 3676- 3677-',
+            },
+            'spike': {
+                'aa': '69- 70- 144- 501Y 570D 681H 716I 982A 1118H',
+            },
+            'orf8': {
+                'aa': 'Q27* 52I 73C',
+            },
+            'n': {
+                'aa': '3L 235F',
+            }
+        }
+    }
+}
+```
+
+### describe-site.py
+
+Will print information about a given location (a "site") in the genome,
+showing you what's in the reference and in the genome you (optionally)
+pass.
+
+In the simplest case, just give a 1-based site and you'll see what's in the
+reference:
+
```

Comment:
Maybe worth saying here that the offsets that are output are 0-based. Unless I'm mistaken.

Reply:
Ah, it says below, sorry, disregard...